### PR TITLE
Fix: aptos editor types

### PR DESCRIPTION
**Pull request by akira02**
## Summary
I fix the wrong type in sdk, but here is wrong too.

## Related Links

**Asana**:  

**Mockup**:  

## Checklist

- [ ] Pasted Asana or Mockup link.
- [ ] Tagged labels.

### CODE REVIEW

File: src/components/AptosEditor.tsx [hunk 1]
```diff
@@ -36,7 +37,7 @@ const formatTransactionArgs = (args: Arg[] | undefined) => {
           : currentValue.value;
     }
     return initial;
-  }, {});
+  }, {}) as SignMessagePayload;
```
Comment (mordochi) [inline src/components/AptosEditor.tsx:40]: Did you get the ts error on this file? Somehow I cannot see it on my side even though I just re-start my VSCode 🤔

File: src/types/ScriptTypes.ts [hunk 2]
```diff
@@ -48,3 +48,11 @@ export type AptosScriptAbiKeys =
   | "generic_type_params"
   | "params"
   | "return";
+
+export interface SignMessagePayload {
```
Comment (mordochi) [inline src/types/ScriptTypes.ts:52]: I think we can just export the type from `@blocto/sdk` and we can just import the type from it. I think the experience will be better for developers using Blocto SDK

File: src/components/AptosEditor.tsx [hunk 3]
```diff
@@ -55,7 +56,7 @@ const AptosEditor = (): ReactJSXElement => {
   }, []);
 
   const handleSignMessage = useCallback((args) => {
-    return new Promise<void>((resolve) => {
+    return new Promise<unknown>((resolve) => {
```
Comment (mordochi) [inline src/components/AptosEditor.tsx:59]: Should be `Promise<SignMessageResponse>`

**Review by mordochi — APPROVED** [review]

File: src/components/AptosEditor.tsx (hunk 1 above)
Comment (akira02) [reply src/components/AptosEditor.tsx:40]: Will appear after updated to 0.4.7-beta.0 with type correction. But moojing is also changing so I only commit type update here.

File: src/types/ScriptTypes.ts (hunk 2 above)
Comment (akira02) [reply src/types/ScriptTypes.ts:52]: ok

File: src/components/AptosEditor.tsx (hunk 1 above)
Comment (mordochi) [reply src/components/AptosEditor.tsx:40]: Got it. Thanks

**Pull request merged**